tests/ui/parser/bad-let-else-statement.rs

```diff
@@ -3,8 +3,7 @@
 #![feature(explicit_tail_calls)]
 
 fn a() {
-    let foo = {
-        //~^ WARN irrefutable `let...else` pattern
```

Comment:
Did other errors in this file kill these suggestions?

Reply:
I removed them intentionally in https://github.com/rust-lang/rust/commit/f4931437c249ed5275ae20eb4ac58e3e4484fa98 because 48% of tests/ui/parser/bad-let-else-statement.stderr was these warnings unrelated to the thing that this file is trying to test (the parser), which seemed excessive.

The "irrefutable let...else pattern" warning is already tested independently by tests/ui/let-else/let-else-irrefutable.rs.